packages/material-ui/src/Popover/Popover.js

```diff
@@ -62,7 +62,13 @@ function getScrollParent(parent, child) {
 }
 
 function getAnchorEl(anchorEl) {
-  return typeof anchorEl === 'function' ? anchorEl() : anchorEl;
+  if (typeof anchorEl === 'function') {
+    return anchorEl();
+  }
+  if (typeof anchorEl === 'object' && anchorEl && anchorEl.current) {
+    return anchorEl.current;
+  }
+  return anchorEl;
```

Comment:
This will return the original `React.createRef()` if `current` is not set yet.

Reply:
right, the check has bot tweaked to to `'current' in anchorEl`